lib/wizard.js

```diff
@@ -2,10 +2,22 @@
 
 const express = require('express');
 const _ = require('lodash');
-const FormController = require('hof-controllers').base;
+const FormController = require('hof-form-controller');
+const mix = require('mixwith').mix;
 
 let count = 0;
 
+const createController = (SuperClass, behaviours) => {
+  /*
+   * This class declaration could be better written using
+   * array spread syntax, supported in node >= 5.11.0:
+   *
+   * class Controller extends mix(SuperClass).with(...behaviours) {}
+   */
+  let _mix;
+  return class extends (_mix = mix(SuperClass)).with.apply(_mix, behaviours) {};
```

Comment:
This feels a little like "one line for the sake of being one line". Looking at it, I think this is equivalent:

```js
const _mix = mix(SuperClass);
return class extends _mix.with.apply(_mix, behaviours) {};
```